src/main/java/it/pagopa/interop/probing/eservice/registry/updater/util/RestClient.java

```diff
@@ -26,18 +26,21 @@ public static RestClient getInstance() throws IOException {
   }
 
   private static final String ESERVICE_OPERATION_URL = "api.operations.baseUrl";
+  private static final String ESERVICE_BASE_PATH = "api.eservice.basePath";
```

Comment:
No free string in every file these string have to be centralized in a properly fashion